medical_prescription/tests/test_medical_prescription_order_line.py

```diff
@@ -43,3 +44,18 @@ def test_name_search_none(self):
             len(recs),
             'Length of recs (%d) not equal to 3' % (len(recs)),
         )
+
+    def test_prescription_order_id_required(self):
```

Comment:
My intention here was mostly to test that the field is properly set up as required, not that the ORM is doing its job. Do you think there's no value in that?